### PR TITLE
Feature/185 mobile navigation click outside

### LINKED ISSUE #185: Mobile navigation should close when clicking outside of navigation
We also need to add the toggleBodyClass for when clicking outside businesspartnermenu

### PR DESCRIPTION
**Pull request by dami-gg**
Fixes #185

### CODE REVIEW

File: src/app/components/user/user-menu/user-menu.controller.js [hunk 1]
```diff
@@ -34,6 +35,10 @@ class UserMenuController {
   logout() {
     this.scAuthenticationService.logout();
   }
+
+  hideUserMenu() {
```
Comment (frontendherodk) [inline src/app/components/user/user-menu/user-menu.controller.js]: I think is missing a test

**Comment by frontendherodk** [comment]
Everything looks good! 
Tests are passing and it seems you've covered all the scenarios and edge cases. Nice job!

File: src/app/components/user/user-menu/user-menu.controller.js (hunk 1 above)
Comment (dami-gg) [inline src/app/components/user/user-menu/user-menu.controller.js]: true, added

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:
